packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.classNames.ts

```diff
@@ -1,28 +1,123 @@
 import { memoizeFunction } from '../../Utilities';
-import { mergeStyleSets, ITheme } from '../../Styling';
-import { IContextualMenuStyles } from './ContextualMenu.Props';
-
+import { ITheme, mergeStyles } from '../../Styling';
+import { IContextualMenuStyles, IMenuItemStyles } from './ContextualMenu.Props';
+import { getStyles as getContextualMenuStyles, getMenuItemStyles } from './ContextualMenu.styles';
 export interface IContextualMenuClassNames {
-  title?: string;
+  container: string;
+  root: string;
+  list: string;
+  divider: string;
+  header: string;
+  title: string;
+}
+
+export interface IMenuItemClassNames {
+  item: string;
+  root: string;
+  linkContent: string;
+  icon?: string;
```

Comment:
Why are some of the class names optional?

Reply:
My mistake. I will make all of them optional, to keep consistency and allow for customization.